src/Security/Authentication/Facebook/src/FacebookDefaults.cs

```diff
@@ -24,16 +24,16 @@ public static class FacebookDefaults
         /// <remarks>
         /// For more details about this endpoint, see https://developers.facebook.com/docs/facebook-login/manually-build-a-login-flow#login.
         /// </remarks>
-        public static readonly string AuthorizationEndpoint = "https://www.facebook.com/v8.0/dialog/oauth";
+        public static readonly string AuthorizationEndpoint = "https://www.facebook.com/v11.0/dialog/oauth";
 
         /// <summary>
         /// The OAuth endpoint used to retrieve access tokens.
         /// </summary>
-        public static readonly string TokenEndpoint = "https://graph.facebook.com/v8.0/oauth/access_token";
+        public static readonly string TokenEndpoint = "https://graph.facebook.com/v11.0/oauth/access_token";
 
         /// <summary>
         /// The Facebook Graph API endpoint that is used to gather additional user information.
         /// </summary>
-        public static readonly string UserInformationEndpoint = "https://graph.facebook.com/v8.0/me";
+        public static readonly string UserInformationEndpoint = "https://graph.facebook.com/v11.0/me";
```

Comment:
Amusing v8 -> v11 since we last did this?

Reply:
They rev the version about every 6 months, and we did the update a little earlier in 5.0.